sources/packages/api/src/services/education-program/education-program.service.models.ts

```diff
@@ -16,17 +16,9 @@ export class EducationProgramsSummary {
   credentialType: string;
   approvalStatus: string;
   totalOfferings: number;
+  credentialTypeToDisplay: string;
 }
-
-export class EducationProgramModel {
-  id: number;
-  name: string;
-  description: string;
-  credentialType: string;
-  cipCode: string;
-  nocCode: string;
-  sabcCode: string;
-  approvalStatus: string;
-  programIntensity: ProgramIntensity;
-  institutionProgramCode: string;
+export class EducationProgramsSummaryPaginated {
```

Comment:
As mentioned earlier, I believe that we can create a generic interface to avoid creating the type over and over again.

Reply:
updated as discussed